runtime/litmus/src/lib.rs

```diff
@@ -866,16 +867,19 @@ construct_runtime! {
 
 		// XCM helpers
 		XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>} = 50,
-		PolkadotXcm: pallet_xcm::{Pallet, Call, Event<T>, Origin, Config} = 51,
+		PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin, Config} = 51,
 		CumulusXcm: cumulus_pallet_xcm::{Pallet, Call, Event<T>, Origin} = 52,
 		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 53,
+		XTokens: orml_xtokens::{Pallet, Call, Storage, Event<T>} = 54,
+		Tokens: orml_tokens::{Pallet, Call, Storage, Event<T>} = 55,
 
 		// Litmus pallets
 		ChainBridge: pallet_bridge::{Pallet, Call, Storage, Event<T>} = 60,
 		BridgeTransfer: pallet_bridge_transfer::{Pallet, Call, Event<T>, Storage} = 61,
 		Drop3: pallet_drop3::{Pallet, Call, Storage, Event<T>} = 62,
 		ExtrinsicFilter: pallet_extrinsic_filter::{Pallet, Call, Storage, Event<T>} = 63,
-		IdentityManagement: pallet_identity_management::{Pallet, Call, Storage, Event<T>} = 64,
+		AssetManager: pallet_asset_manager::{Pallet, Call, Storage, Event<T>} = 64,
+		IdentityManagement: pallet_identity_management::{Pallet, Call, Storage, Event<T>} = 65,
```

Comment:
I believe the index number of `IdentityManagement` is used by @suinuj in TEE tests, so we'd better shift AssetManager after it

Reply:
I have swapped the pallet index accordingly, and save 64 on Moonbase for identity pallet too.
However, if there will be a lot of pallets in future, I suggest we start a individual pallet_prefix for such TEE related pallet.